src/test/java/org/myrobotlab/test/AbstractTest.java

```diff
@@ -14,6 +17,7 @@
 import org.junit.Rule;
 import org.junit.rules.TestName;
 import org.myrobotlab.framework.Platform;
+import org.myrobotlab.framework.interfaces.Attachable;
```

Comment:
seems like only services are attachable.. this attachable stuf shouldn't be in the base AbstractTest class.. it should be up one layer in AbstractServiceTest.

Reply:
I'm not sure if only Services are attachable...